packages/react-native-gesture-handler/src/web/handlers/GestureHandler.ts

```diff
@@ -504,7 +531,7 @@ export default abstract class GestureHandler implements IGestureHandler {
 
     const trackerData = this.tracker.trackedPointers;
 
-    if (trackerData.size === 0) {
+    if (trackerData.size === 0 || !this.propsRef) {
```

Comment:
We could check this at the beginning, before measuring the view. Maybe it doesn't boost performance that much, but I can see that you tend to check it at the beginning of the functions so I think it will be better to stick to one convention.